spec/javascripts/govuk_publishing_components/analytics/explicit-cross-domain-links.spec.js

```diff
@@ -93,6 +93,31 @@ describe('Explicit cross-domain linker', function () {
         expect(element.attr('href')).toEqual('/somewhere?cookie_consent=accept')
       })
     })
+
+    describe('user has interacted with the cookie banner on the current page', function () {
+      it('modifies the link href to append cookie_consent parameter "accept" if the cookie-consent event was fired', function () {
+        GOVUK.cookie('cookies_preferences_set', null)
+        explicitCrossDomainLinks.start(element)
+
+        GOVUK.cookie('cookies_preferences_set', 'true')
+        GOVUK.setConsentCookie({ usage: true })
+        window.ga = undefined
+        window.GOVUK.triggerEvent(window, 'cookie-consent')
+
+        expect(element.attr('href')).toEqual('/somewhere?cookie_consent=accept')
+      })
+
+      it('modifies the link href to append cookie_consent parameter "reject" if the cookie-reject event was fired', function () {
+        GOVUK.cookie('cookies_preferences_set', null)
+        explicitCrossDomainLinks.start(element)
+
+        GOVUK.cookie('cookies_preferences_set', 'true')
```

Comment:
These three lines appear to be duplicated in the previous test - could they be moved into a `beforeEach` function for this `describe` block?

Reply:
Great point, have updated as suggested.